src/main/java/duke/Task.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+/**
+ * Represents a task given by the user, containing the name and the completion status of the task.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+    
+    /**
+     * Creates a new task with the specified description.
+     * @param description The description of the task.
```

Comment:
Perhaps you should leave a line between your description and your @param line?

Reply:
Alright thanks